pkg/storage/gcp/getter_test.go

```diff
@@ -0,0 +1,3 @@
+package gcp
+
+// see gcp_test.go for a round-trip test that subsumes tests for the getter
```

Comment:
is it possible to use package for storage test with GCP?

Reply:
It will be, I want to get this in and then start on refactoring to be able to use a mock in tests. That issue, #263, might include moving to the shared package. I'm not sure yet. Maybe I will make two to keep them shorter. This one got big.

Either way I began to move it over after #215 went in but ran into issue cleaning up the resources after tests so decided to work on mocks first.

Reply:
great 👍  for shorter PR, i would like to achieve a lot of smaller PRs which can be reviewed very fast

Reply:
me too, I think we will be able to move a lot quicker over time this way.